src/components/Select/ComboboxMulti/ComboboxMulti.tsx

```diff
@@ -0,0 +1,112 @@
+import * as React from 'react';
+import { IoIosArrowUp, IoIosArrowDown } from 'react-icons/io';
+import bem from '../../../utils/bem/bem';
+import { Props as SelectBaseProps, SelectBase } from '../SelectBase';
+import { SuggestionsList } from '../SuggestionsList';
+import styles from './ComboboxMulti.scss';
+import { ESCAPE_KEY, TAB_KEY } from '../../../constants';
+
+interface Props<S>
+    extends Omit<
+        Omit<Omit<SelectBaseProps<S>, 'listRenderer'>, 'focusedRenderer'>,
+        'blurredRenderer'
+    > {
+    inputPlaceholder: string;
+}
+
+const { elem } = bem('ComboboxMulti', styles);
+
+export function ComboboxMulti<S>(props: Props<S>) {
+    const {
+        onSelectionChange,
+        inputRef: inputRefFromProps,
+        suggestions,
+        suggestionToString,
+        showClearButton,
+        noSuggestionsPlaceholder,
+        onBlur,
+        onInputValueChange,
+        onClearAllSelected,
+        inputPlaceholder,
+        ...rest
+    } = props;
+    const inputRef = inputRefFromProps || React.createRef<HTMLInputElement>();
+
+    // eslint-disable-next-line react/display-name
+    const renderFocused = ({ getInputProps, getToggleButtonProps, onBlur: blur }) => {
+        const handleInputKeyDown = event => {
+            if (event.key === TAB_KEY) {
+                inputRef.current?.blur();
+                blur();
+            } else if (event.key === ESCAPE_KEY) {
+                // prevents key propagation and sets the focus on parent component
```

Comment:
I'm not sure if setting the focus on the parent is correct behaviour here. The focus would still be in the component, so it would still be open. Can you clarify the requirements about it?

Reply:
@eszthoff once the focus was removed from the input, the app doesn't handle events like key press anymore until you set it somewhere by your own. Just to make it more convenient I decided to move the focus to the parent element. For now it's used only in Autosuggest of the LocationSelector component. The issue behind it was impossibility of closing the modal window by pressing on escape after closing the suggestions dropdown was closed by the same escape press.